wagtail/contrib/modeladmin/templates/modeladmin/edit.html

```diff
@@ -11,9 +11,11 @@
 
         {% if user_can_delete %}
             <div class="dropdown-toggle icon icon-arrow-up"></div>
-            <ul role="menu">
-                <li><a href="{{ view.delete_url }}" class="shortcut">{% trans "Delete" %}</a></li>
-            </ul>
+            <nav aria-label="actions">
+                <ul>
+                    <li><a href="{{ view.delete_url }}" class="shortcut">{% trans "Delete" %}</a></li>
```

Comment:
There’s only one button here, so I’d suggest dropping the `nav` landmark here as well.

Reply:
removing